aten/src/ATen/core/Array.h

```diff
@@ -23,10 +23,16 @@ struct Array {
   C10_HOST_DEVICE Array() = default;
   C10_HOST_DEVICE Array(const Array&) = default;
   C10_HOST_DEVICE Array& operator=(const Array&) = default;
+  C10_HOST_DEVICE Array(Array&&) = default;
```

Comment:
They shouldn't be noexcept?

Reply:
Can be, but this class should be replaced by std::array once cuda 12.1 is discard.